docs/architecture/adr-062-p2p-architecture.md

```diff
@@ -469,31 +575,9 @@ func EchoReactor(ctx context.Context, channel *p2p.Channel, peerUpdates p2p.Peer
 }
 ```
 
-### Implementation Plan
-
-The existing P2P stack should be gradually migrated towards this design. The easiest path would likely be:
-
-1. Implement the `Channel` and `PeerUpdates` APIs as shims on top of the current `Switch` and `Peer` APIs, and rewrite all reactors to use them instead.
-
-2. Port the `privval` package to no longer use `SecretConnection` (e.g. by using gRPC instead), or temporarily duplicate its functionality.
-
-3. Rewrite the current MConn connection and transport code to use the new `Transport` API, and migrate existing code to use it instead.
-
-4. Implement the new `peer` and `peerStore` APIs, and either make the current address book a shim on top of these or replace it.
-
-5. Replace the existing `Switch` abstraction with the new `Router`.
-
-6. Move the PEX reactor and other address advertisement/exchange into the P2P core, possibly the `Router`.
-
-7. Consider rewriting and/or cleaning up reactors and other P2P-related code to make better use of the new abstractions.
-
-A note on backwards-compatibility: the current MConn protocol takes whole messages expressed as byte slices and splits them up into `PacketMsg` messages, where the final packet of a message has `PacketMsg.EOF` set. In order to maintain wire-compatibility with this protocol, the MConn transport needs to be aware of message boundaries, even though it does not care what the messages actually are. One way to handle this is to break abstraction boundaries and have the transport decode the input's length-prefixed message framing and use this to determine message boundaries, unless we accept breaking the protocol here.
-
-Similarly, implementing channel handshakes with the current MConn protocol would require doing an initial connection handshake as today and use that information to "fake" the local channel handshake without it hitting the wire.
-
 ## Status
 
-Accepted
+Partially implemented
```

Comment:
Maybe link to the tracking issue (or WIP google doc?) here?

Reply:
Good idea, thanks.